shared/build.gradle.kts

```diff
@@ -78,6 +80,7 @@ kotlin {
         implementation(Deps.AndroidXTest.runner)
         implementation(Deps.AndroidXTest.rules)
         implementation(Deps.Coroutines.test)
+        implementation(Deps.ktor.androidCore)
```

Comment:
This isn't needed anymore since it's in androidMain
```suggestion
```

Reply:
Gotcha, removed